packages/web3-utils/test/unit/formatter.test.ts

```diff
@@ -827,6 +847,32 @@ describe('formatter', () => {
 
 				expect(result).toEqual(expected);
 			});
+
+			it('should format object with oneOf', () => {
```

Comment:
can you also create a test using oneOf when the `to` property is undefined?

Reply:
Hi, Could you give me an example of this one?  Do you mean something like this?
```
const data ={
    from: '0x7ed0e85b8e1e925600b4373e6d108f34ab38a401'
};

const result = { from: '0x7ed0e85b8e1e925600b4373e6d108f34ab38a401'};
```

Reply:
yes